src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
```

Comment:
The class `FigureSupplier` should not begin with an empty line according to the checklist.